pkg/controller/kfservice/kfservice_controller.go

```diff
@@ -65,16 +63,15 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 	}
 
 	// Watch for changes to KFService
-	err = c.Watch(&source.Kind{Type: &servingv1alpha1.KFService{}}, &handler.EnqueueRequestForObject{})
+	err = c.Watch(&source.Kind{Type: &kfservingv1alpha1.KFService{}}, &handler.EnqueueRequestForObject{})
```

Comment:
You need to watch both right?

Reply:
if you scroll down, I do watch both, diff tricks you :)